README.md

```diff
@@ -118,6 +118,51 @@ or check the [PaaS documentation](https://docs.genymotion.com/paas/01_Requiremen
 </script>
 ```
 
+## Style and CSS
+
+The player use css variables to stylized some parts of the app, but the player is still totally customisable throught overload of the css class.
```

Comment:
```suggestion
The player use css variables to stylize some parts of the app, but the player is still totally customisable through overloading the css classes.
```

Reply:
Also: "The player uses" (instead of "The player use").

"style" might also be more straightforward than "stylize".